oneview/resource_volume.go

```diff
@@ -287,9 +291,8 @@ func resourceVolumeUpdate(d *schema.ResourceData, meta interface{}) error {
 	isPermanent := d.Get("is_permanent").(bool)
 	isShareable := d.Get("is_shareable").(bool)
 
-	properties := d.Get("properties").(*schema.Set).List()[0].(map[string]interface{})
 	volume.Name = d.Get("name").(string)
-	volume.ProvisioningTypeForUpdate = properties["provisioning_type"].(string)
+	volume.ProvisioningTypeForUpdate = d.Get("provisioning_type").(string)
```

Comment:
"provisioning_type" is under Properties Struct, why are we accessing using d.Get here and I see no such change or documentation regarding this in example?

Reply:
provisioning_type also there in the volume struct. We can use it from there.